lockbox-ios/Action/FxAAction.swift

```diff
@@ -38,6 +63,39 @@ enum FxAError: Error {
     case Unknown
 }
 
+extension FxAError: TelemetryAction {
```

Comment:
The FxA error never gets sent as an `Action`, so this extension won't ever get invoked. I think we may need to circle back with @jhugman about how to do this logging in our current "old"-FxA world.